crates/autopilot/src/domain/settlement/mod.rs

```diff
@@ -41,55 +128,22 @@ impl Settlement {
         }
 
         let auction = persistence.get_auction(settled.auction_id).await?;
+        let database_orders = persistence.orders_that_exist(&settled.order_uids()).await?;
```

Comment:
This function 🙈 

Can we not get the orders that exist from the auction we just fetched in the line above?

Reply:
It's needed to differentiate between `jit orders` and `user orders settled out of Auction`.

Reply:
Now that we got rid of the `user orders settled out of Auction` concept, can we get rid of this function as well?

Reply:
removed